docs/getting-started/connect-widget.mdx

```diff
@@ -13,17 +13,248 @@ Here's a preview:
   width="90%"
   height="700"
   src="https://connect.metriport.com/?token=demo"
-></iframe>
-<br />
+  ></iframe>
+
+
+## How to Implement
+
+You can use the references below for sample implementations on various platforms. Generally, you can simply link to the Connect Widget in web apps, and use WebViews to implement the Widget on mobile platforms.
+
+<Info>
+  As per the [Quickstart guide](/getting-started/connect-quickstart#4-link-to-the-metriport-connect-widget-in-your-app), you'll need to generate a temporary token for each Connect Widget session.
+</Info>
```

Comment:
This was done before, but it would be nice to add the details about the session duration, as [documented on the quickstart](https://github.com/metriport/metriport/blob/develop/docs/getting-started/connect-quickstart.mdx?plain=1#L138).

Maybe even improving a bit on the text - now that I read that part about being reused it doesn't sound super clear.